src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_evp_cipher.c

```diff
@@ -134,40 +134,31 @@ CipherCtx* CryptoNative_EvpCipherCreatePartial(intptr_t type)
     CipherCtx* ctx = malloc(sizeof(CipherCtx));
     ctx->cipher = cipher;
     ctx->type = type;
+    ctx->tagLength = TAG_MAX_LENGTH;
     ctx->ivLength = 0;
     ctx->encMode = 0;
     ctx->key = NULL;
     ctx->iv = NULL;
-    memset(ctx->tag, 0, TAG_MAX_LENGTH);
     return CheckJNIExceptions(env) ? FAIL : ctx;
 }
 
-int32_t CryptoNative_EvpCipherSetKeyAndIV(CipherCtx* ctx, uint8_t* key, uint8_t* iv, int32_t enc)
+int32_t CryptoNative_EvpCipherSetTagLength(CipherCtx* ctx, int32_t tagLength)
```

Comment:
I believe @elinor-fung used "AndroidCryptoNative_" as the prefix on new exports she added.

Since the Interop.cs file isn't shared the export name shouldn't matter, so this can use the right export prefix for this library already.

Reply:
(Maybe this is just that some functions got reordered, so maybe the comments don't apply)